lib/src/player.dart

```diff
@@ -285,6 +277,19 @@ class Player with PlayerEventHandler implements PlayerApi {
 
   @override
   AnalyticsApi get analytics => _AnalyticsApi(this);
+
+  @override
+  Future<bool> get isCastAvailable => _invokeMethod(Methods.isCastAvailable);
+
+  @override
+  Future<bool> get isCasting => _invokeMethod(Methods.isCasting);
+
+  @override
+  Future<void> castVideo() async => _invokeMethod<void>(Methods.castVideo);
+
+  @override
+  Future<void> castStop() => _invokeMethod<void>(Methods.castStop);
```

Comment:
Let's declare all of them async

Reply:
Removed the async instead as it is not needed: 69c6b2ae80bdd5e50591c8acc7a8f8765d56e00c